.github/workflows/comment_contributing.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: Comment on the pull request
+
+on:
+  workflow_call:
+
+jobs:
+  comment-on-pr:
+    name: Comment on PR
+    runs-on: ubuntu-22.04
+    if: github.event.workflow_run.event == 'pull_request'
```

Comment:
I think we can probably remove this condition and use the `pull_request_target` `open` event to trigger this workflow. This way, it will only create one comment per pull request. Additionally, we should probably only comment on pull requests from forks.

```
on:
  pull_request_target:
    types: [opened]

jobs:
  comment-contributing-guide:
    if: ${{ github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name }}
    uses: canonical/operator-workflows/.github/workflows/comment_contributing.yaml@main
    secrets: inherit
```

Reply:
nvm, I think the comment above is outside the scope of this pull request, sorry.

Reply:
I think the `github.event.workflow_run.event == 'pull_request'` condition exists because the test workflow used to run in the `push` event as well (for publish charms). But that's probably not needed anymore.

Reply:
Mmm makes sense. Updated